src/icon/legRest.tsx

```diff
@@ -0,0 +1,17 @@
+// tslint:disable:max-line-length
+import React from 'react'
+import StatusIcon from '_utils/icon/status'
+
+export const LegRest = (props: Icon) => (
```

Comment:
Are you sure you need them to extends `StatusIcon`, not the `BaseIcon`?

Reply:
Yes all of them need to be on or off, so that we know if a amenity is available or not.